File: C++/Day 12/Shape/circle.cpp
#include "shape.h"

void Circle::Accept(){
	cout<<"Enter value of radius: ";
	cin>>radius;
}

float Circle::Area(){
	float area = 3.14*(radius*radius);
	return area;
}

float Circle:Peri(){
	float peri = 2 * 3.14 * radius;
	return peri;
}

File: Java/Day 7/Shape/src/com/cdac/acts/Circle.java
package com.cdac.acts;

public class Circle extends Shape {
	private double radius;
	private final double PI = 3.14;
	
	public Circle() {
		this.radius = 0.0;
	}
	
	public Circle(double radius) {
		this.radius = radius;
	}
	
	public void setRadius(double radius) {
		this.radius=radius;
	}
	
	public double calculateArea() {
		double result = PI * radius * radius;
		return result;
	}
	
	public double calculatePerimeter() {
		double result = 2 * PI * radius;
		return result;
	}
	
	public void draw(){
		System.out.println("No need to draw real Circle");
	}
	
	public String toString() {
		return "Circle [ Area = " + calculateArea() + "Perimeter = " + calculatePerimeter() + "]";
	}
	
	

}
